### DISCUSSION TITLE
Plotting pole figure problem

### DISCUSSION THREAD

**Discussion by WeiHuang0722** [mtex-toolbox/mtex#1746] [Ask Anything]
Hi all,
I came across a problem when I tried to plot pole figure. I have csc and txt files which include three euler angles and a weight parameter for ploting pole figures. However, when I used the import wizard small window to import the file under the tag of EBSD, assigning the correct header. Then, I clicked finish button, but it always shows me an error dialog window saying "Could not detect the file format. You may ask for help in the MTEX forum..." Thus, I am currently not able to use MTEX for ploting pole figures....
Does anyone know how to tackle this problem?
Thanks a lot!!!

**Comment by lfgmorales** [comment]
Hi

Two things:

1) importing pole figures aa EBSD will not work, on import_wizard you should use the “pole figure” panel

2) I think import wizard will not work with *.csc files because it is an uknown format for pole figures in mtex but you may want to try your *.txt file, if the format is it should work. But you can also send an example dataset, so people can incorporate on import_wizard

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Feb 2023, at 03:56, Wei Huang ***@***.***> wrote:

﻿

Hi all,
I came across a problem when I tried to plot pole figure. I have csc and txt files which include three euler angles and a weight parameter for ploting pole figures. However, when I used the import wizard small window to import the file under the tag of EBSD, assigning the correct header. Then, I clicked finish button, but it always shows me an error dialog window saying "Could not detect the file format. You may ask for help in the MTEX forum..." Thus, I am currently not able to use MTEX for ploting pole figures....
Does anyone know how to tackle this problem?
Thanks a lot!!!

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1746>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN5XROZ4NT6FKOXS2GTWZAPLXANCNFSM6AAAAAAVGMD74I>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1746", "url": "https://github.com/mtex-toolbox/mtex/discussions/1746", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by WeiHuang0722** [reply]
Thanks for the clarification. However, I just have Euler Angles for plotting, without the data for pole figure panel...so the small window won't work for this situation...maybe I have to write some codes for the purpose of plotting.
